lib/gpu_memory_service/tests/test_cli_server.py

```
@@ -0,0 +1,200 @@
+        str(tmp_path / "kv_cache.sock"),
+    ]
+    configs = [
+        Config(
+            device=0,
+            tag=tag,
+            socket_path=socket_path,
+            alloc_retry_interval=0.5,
+            alloc_retry_timeout=60.0,
+            verbose=False,
+        )
+        for tag, socket_path in zip(dual_server.TAGS, socket_paths)
```

Ruff (0.15.20)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)